docs/java-rest/high-level/indices/get_field_mappings.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[java-rest-high-get-field-mappings]]
+=== Get Field Mappings API
+
+[[java-rest-high-get-field-mappings-request]]
+==== Get Field Mappings Request
+
+A `GetFieldMappingsRequest` can have an optional list of indices, optional list of types and the list of fields:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[get-field-mapping-request]
+--------------------------------------------------
+<1> An empty request
+<2> Setting the indices to fetch mapping for
+<3> The types to be returned
+<4> The fields to be returned
+
+==== Optional arguments
+The following arguments can also optionally be provided:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[get-field-mapping-request-indicesOptions]
+--------------------------------------------------
+<1> Options for expanding indices names
+
+[[java-rest-high-get-field-mappings-sync]]
```

Comment:
aren't we missing a couple of supported parameters here?

Reply:
good spotted - I found only one - `local`, pls let me know if anything else you have in mind